turbinia/evidence.py

```diff
@@ -1082,6 +1082,10 @@ def _validate(self):
     except subprocess.CalledProcessError as exception:
       raise TurbiniaException(
           f'Error validating plaso file: {exception!s}') from exception
+    except json.decoder.JSONDecodeError as exception:
```

Comment:
Is this supposed to be in this PR?  Why wouldn't we want to error out in this condition as well?

Reply:
It doesn't have to be in this PR but we've seen a couple times this could potentially fail if the output of the tool changes. I thought it would make sense to "fail open" if the tool runs successfully but we get unexpected output from it. That way the task can try to process the plaso file. It will still error out if the execution of pinfo fails.